release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -156,6 +156,450 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha3](https://githu
+        cert: certPath,
+        protocol: {
+            name: tcp:TLS,
+            versions: ["TLSv1.2", "TLSv1.1"]
+        },
+        ciphers: ["TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA"]
+    });
+
+    string msg = "Hello Ballerina Echo from secure client";
+    byte[] msgByteArray = msg.toBytes();
+    check socketClient->writeBytes(msgByteArray);
+
+    readonly & byte[] receivedData = check socketClient->readBytes();
+
+    check socketClient->close();
+}
+```
+
+```ballerina
+configurable string keyPath = ?;
+configurable string certPath = ?;
+
+service on new tcp:Listener(9002, secureSocket = {
+       key: {
+        certFile: certPath,
+        keyFile: keyPath
+    },
+    protocol: {
+        name: tpc:TLS,
+        versions: ["TLSv1.2", "TLSv1.1"]
+    },
+    ciphers: ["TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA"]
+}) {
+
+    isolated remote function onConnect(tcp:Caller caller) returns tcp:ConnectionService {
+        io:println("Client connected to secureEchoServer: ", caller.remotePort);
+        return new EchoService(caller);
+    }
+}
+
+service class EchoService {
+  
+    remote function onBytes(readonly & byte[] data) returns (readonly & byte[])|tcp:Error? {
+        io:println("Echo: ", 'string:fromBytes(data));
+        return data;
+    }
+}
+```
+
+###### Included a `tcp:Caller` as an optional parameter in the `onBytes()` method.
+```
+service class EchoService {
+  
+    remote function onBytes(tcp:Caller caller, readonly & byte[] data) returns (readonly & byte[])|tcp:Error? {
+        io:println("Echo: ", 'string:fromBytes(data));
+        check caller->writeBytes(data);
+    }
+}
+```
+
+##### Kafka Package Updates
+
+###### Renamed the `sendProducerRecord` function in the client object `Producer` to `send`.
+
+###### Renamed the `flushRecords` function in the client object `Producer` to `’flush`.
+
+###### Replaced the `kafka:ConsumerError` and `kafka:ProducerError` with `kafka:Error`.
+
+##### NATS Package Updates
+
+###### Renamed the `ConnectionConfig` record to `ConnectionConfiguration`. 
+
+###### Included `url` as a field in the `ConnectionConfiguration` record. 
+
+###### Changed the `ConnectionConfiguration` in the client and listener init functions to an included record parameter. This allows the record field values to be passed as named parameters. 
+
+##### STAN Package Updates
+
+###### Renamed the `StreamingConfig` record to `StreamingConfiguration`. 
+
+###### Included the `url` as a field in the `StreamingConfiguration` record. 
+
+###### Changed the `StreamingConfiguration` in the client and listener init functions to an included record parameter. This allows the record field values to be passed as named parameters. 
+
+##### RabbitMQ Package Updates
+
+###### Renamed the `ConnectionConfig` record to `ConnectionConfiguration`. 
+
+###### Common Standard Library Updates
+
+###### All the timeout configurations are converted to accept decimal values and the time unit is in seconds.
```

Comment:
If we don't have content under these can we make them bullet points? @praneesha wdyt?

Reply:
I marked them all as bullet points with this.
https://github.com/ballerina-platform/ballerina-release/pull/115